specification/cognitiveservices/OpenAI.Inference/models/completions.common.tsp

```diff
@@ -35,3 +37,56 @@ enum CompletionsFinishReason {
     """)
   contentFiltered: "content_filter",
 }
+
+@added(ServiceApiVersions.v2023_06_01_Preview)
+@doc("Ratings for the intensity and risk level of harmful content.")
+enum ContentFilterSeverity {
+  @doc("General content or related content in generic or non-harmful contexts.")
+  safe: "safe";
+
+  @doc("Harmful content at a low intensity and risk level.")
+  low: "low";
+
+  @doc("Harmful content at a medium intensity and risk level.")
+  medium: "medium";
+
+  @doc("Harmful content at a high intensity and risk level.")
+  high: "high";
+}
+
+@added(ServiceApiVersions.v2023_06_01_Preview)
+@doc("Information about filtered content severity level and if it has been filtered or not.")
+model ContentFilterResult {
+  @doc("")
+  @projectedName("json", "severity")
+  severity: ContentFilterSeverity;
+
+  @doc("")
+  @projectedName("json", "filtered")
+  filtered: boolean;
+
+}
+
+@added(ServiceApiVersions.v2023_06_01_Preview)
+@doc("Information about the content filtering category, if it has been detected.")
+model ContentFilterResults {
+  sexual: ContentFilterResult;
+  violence: ContentFilterResult;
+  hate: ContentFilterResult;
+  selfHarm: ContentFilterResult;
+  //error?
```

Comment:
I think you probably don't have to add error here. That would be a variant that will be taken care for you. Unless it's a custom thing.